src/components/CarouselItem/CarouselItem.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <!-- add active class based on index -->
+  <div
+    class="image fade"
+    :class="index == active ? 'image-active' : ''"
```

Comment:
Consider passing down a `isActive` property, which already has the resolved fact of whether the item is active in turn minimising the need for two props. 😉